src/components/Pillbox/Pillbox.js

```diff
@@ -134,12 +135,14 @@ export default class Pillbox extends Component {
           ref={this.list} />
         }
       </div>}
-      <ReactTooltip place='top'
-        type='dark'
-        id='pill-label'
-        effect='solid'
-        disable={!editable}
-        delayShow={500} />
+      {!isMobile.any && (
```

Comment:
i wonder if we should just turn off all tooltips on mobile through a custom Tooltip component?

Reply:
Maybe, probably. Don't want to take that on as part of this scope right now.